ta4j/pom.xml

```diff
@@ -1,11 +1,11 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <groupId>eu.verdelhan</groupId>
+        <groupId>org.ta4j</groupId>
         <artifactId>ta4j-parent</artifactId>
         <version>0.10-SNAPSHOT</version>
     </parent>
-    <artifactId>ta4j</artifactId>
+    <artifactId>core</artifactId>
```

Comment:
Should rather be "ta4j-core", right?

Reply:
Is it bad practice to have a generic `artifactId` like `core` if the `groupId` sufficiently describes where it belongs?

Reply:
Maybe not. But it is not common. See:
```
<dependency>
    <groupId>org.apache.commons</groupId>
    <artifactId>commons-math3</artifactId>
    <version>3.6.1</version>
</dependency>
```
or:
```
<dependency>
    <groupId>com.google.code.gson</groupId>
    <artifactId>gson</artifactId>
    <version>2.8.1</version>
</dependency>
```

Reply:
And it seems weird if the parent is still "ta4j-parent".

Reply:
Yes, i think it should not be ta4j-parent in the parent, but ta4j. I like also the suggestion to describe this repository with ta4j-core, because it is more understandable regarding the old description.
so we would have

->org.ta4j
--> org.ta4j.ta4j-core
--> org.ta4j.ta4j-examples

what do you think?

Reply:
I would have kept the parent as "ta4j-parent" because IMO it avoids ambiguity.
We would have:
groupId: org.ta4j
artifactIds:
-> ta4j-parent (i.e. we kept the "parent" word to avoid you directly add it as a dependency)
--> ta4j-core
--> ta4j-examples
--> ta4j-whatever